build/ndm-daemonset/Dockerfile

```diff
@@ -16,10 +16,15 @@
 #
 FROM golang:1.14.7 as build
 
-ARG TARGETPLATFORM
+ARG TARGETOS
+ARG TARGETARCH
+ARG TARGETVARIANT=""
```

Comment:
Why do we need to set the `TARGETVARIANT` as empty string?, while we dont set the other envs.

Reply:
This is because it is only set when it's given armv7 or 8 or equivalent, if we move any additional logic from this variable this provides just a default empty string rather than null value.